### PR TITLE
Update setActive occurrences to state that org slug is accepted

### PR DESCRIPTION
**Pull request by wobsoriano**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1328/references/javascript/clerk/session-methods#set-active-params
> - https://clerk.com/docs/pr/1328/references/react/use-sign-in#set-active-params
> - https://clerk.com/docs/pr/1328/references/react/use-sign-up#set-active-params
> - https://clerk.com/docs/pr/1328/references/react/use-session-list#set-active-params
> - https://clerk.com/docs/pr/1328/references/react/use-organization-list#set-active-params

### This PR:

- Updates all occurrences of `setActive` params to state that an organization slug can be used besides the ID.
- https://github.com/clerk/javascript/pull/3825

### CODE REVIEW

**Review by izaaklauer — APPROVED** [review]
Looks good to me! Let's hold off bringing this out of draft mode until we have functionality merged in clerk/javascript.

**Review by alexisintech — APPROVED** [review]

**Comment by wobsoriano** [comment]
Thanks @alexisintech, updated the description!

**Pull request merged**
